packages/backend/src/services/CsvService/convertSqlToCsv.ts

```diff
@@ -10,9 +10,13 @@ type Args = {
 function run() {
     const { results, customLabels }: Args = workerData;
 
-    convertSqlToCsv(results, customLabels).then((csv) => {
-        if (parentPort) parentPort.postMessage(csv);
-    });
+    convertSqlToCsv(results, customLabels)
+        .then((csv) => {
+            if (parentPort) parentPort.postMessage(csv);
+        })
+        .catch((_error) => {
+            // TODO: not implemented
+        });
```

Comment:
just a catch statement.. we should probably pass along the error message to the process that executes the worker 🤷

Reply:
Should this also be an `await`/`try/catch` instead? In the meantime I'd also consider adding a log statement for the error (unless this is something we know to pollute logs).

Reply:
@rephus thoughts?

Reply:
mm, I think this is the method that runs on a separate thread, I'm not sure what is more compatible with what we currently have. perhaps void + catch ?